Dockerfile

```
@@ -79,7 +79,7 @@ COPY --from=pg-build /home/nonroot/postgres_install.tar.gz /data/
 RUN mkdir -p /data/.neon/ && chown -R neon:neon /data/.neon/ \
     && /usr/local/bin/pageserver -D /data/.neon/ --init \
        -c "id=1234" \
-       -c "broker_endpoints=['http://etcd:2379']" \
+       -c "broker_endpoint='http://127.0.0.1:50051'" \
```

```suggestion
       -c "broker_endpoint='http://storage_broker:50051'" \
```

Should be slightly better for running in docker.

---

I would agree: in terms of Docker, localhost access feels odd, as not that much extra processes are started fluently to listen something.